app/models/user.rb

```diff
@@ -1,12 +1,32 @@
 class User < ApplicationRecord
   enum gender: [:male, :female]
+  attr_accessor :remember_token
```

Comment:
attr_accessor tự động gen ra 2 phương thức getter and setter cho biến remember_token ạ.